packages/statistics/index.rst

```diff
@@ -314,40 +314,62 @@ use the :mod:`scipy.stats` sub-module of `SciPy
 Student's t-test: the simplest statistical test
 ------------------------------------------------
 
-1-sample t-test: testing the value of a population mean
+One-sample tests: testing the value of a population mean
 ........................................................
 
 .. image:: two_sided.png
    :scale: 50
    :align: right
 
-:func:`scipy.stats.ttest_1samp` tests if the population mean of data is
-likely to be equal to a given value (technically if observations are
-drawn from a Gaussian distributions of given population mean). It returns
```

Comment:
This may lead to two misunderstandings. I've adjusted the text here and added the paragraph beginning with "Technically" to avoid these.